applications/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-members = ["contracts/*"]
+members = ["contracts/apps/*", "packages/bundles/wyndex"]
```

Comment:
YESSS All on-chain apps together now 🔥. Don't forget to include "contracts/adapters/*"

Reply:
And not sure about wyndex bundle